shared/oae/bundles/email/default.properties

```diff
@@ -1,6 +1,9 @@
 ACTIVITY_EMAIL_CHANGE_PREFRENCES = You can change your <a href="${url}">email preferences</a> anytime.
 ACTIVITY_EMAIL_GREETING = Hi ${displayName}
 ACTIVITY_EMAIL_SUBJECT_DAILY = Today's activity summary
+ACTIVITY_EMAIL_SUBJECT_INVITE_ACTOR_1 = ${actor1DisplayName} has invited you to collaborate
```

Comment:
Can we always use one of these 3 as the grey title under the logo in the invitation email? When there's a single activity, it's redundant to have "John Norman has invited you to the file "City+of+London+Skyline.jpg" there as well